redhat/release/update-to-head.sh

```diff
@@ -60,7 +60,9 @@ fi
 
 # Update redhat's main and take all needed files from there.
 git fetch origin "$midstream_ref"
-git checkout origin/"$midstream_ref" "$custom_files"
+git checkout origin/"$midstream_ref" $custom_files
+
+echo "HERE"
```

Comment:
I feel like this is just a leftover from testing but outside of this it all looks good!

Reply:
hahah yes it is